### ISSUE TITLE
Achievements update

### ISSUE THREAD

**Issue by Alisonmontteiro** [MAFINS/MenyooSP#235]
Please, in a next update add the release of these achievements **"The Data Breaches, The Bogdan Problem, Getting Started, The Doomsday Scenario, Orbital Obliteration, A World Worth Saving, Masterminds, Elitist"** currently in this version are the only achievements that do not release. Please consider this in a future update.

**Comment by Chris-Azusa** [comment]
Yeah, please add those achivement on menyoo. Love your work but feel incompleted withouth that. Tks
